app/models/applicant.rb

```diff
@@ -25,6 +25,10 @@ def full_name
     "#{first_name} #{last_name}".strip
   end
 
+  def address
+    addresses.first
+  end
```

Comment:
I think using a `has_one` association would be better - as it wouldn't gather all the addresses and then discard all but the first:
```
has_one :address, -> { order(created_at: :desc) }
```

Reply:
I don't think `addresses.first` gather all the address, SQL only fetch the first address with `LIMIT 1`.
But anyway, your `has_one` suggesting is more elegant so I'll do that

Reply:
You're right (as usual), but another problem with `addresses.first` is that it finds the first via `id`, and as addresses use UUIDs as ids, that means it plucks a random address out.